dotnet/src/SemanticKernel.Abstractions/Search/FilterClause.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Search;
+
+/// <summary>
+/// Base class for filter clauses.
+/// </summary>
+[Experimental("SKEXP0001")]
+public class FilterClause
+{
+    /// <summary>
+    /// The type of FilterClause.
+    /// </summary>
+    /// <remarks>
+    /// The <see cref="FilterClause"/> is used to request that the underlying search service should
+    /// filter search results based on the specified criteria.
+    /// </remarks>
```

Comment:
Consider adding these remarks to the class itself, since I think it's generally applicable, not just to the type.